src/networking/NetworkService.js

```diff
@@ -1,17 +1,20 @@
 import axios from 'axios';
+import applyCaseMiddleware from 'axios-case-converter';
 import { baseURL, headers } from '@/networking/config';
 import { resInterceptor } from '@/networking/interceptors';
 
 export class NetworkService {
   constructor() {
-    this.client = axios.create({ baseURL, headers });
+    this.client = applyCaseMiddleware(axios.create({ baseURL, headers }));
     this.client.interceptors.response.use(resInterceptor.onFulfill, resInterceptor.onReject);
   }
 
+  //todas las llamdas le pone el token
   setAccessToken(token) {
     this.client.defaults.headers.common.authorization = `Bearer ${token}`;
   }
 
+  //cada logout
```

Comment:
These debugging notes are awesome to help you debug better and faster 🔥 just don't forget to remove them before pushing your changes.

Reply:
Perfect!